### PR TITLE
Bugfix/loading errors

### PR DESCRIPTION
**Pull request by JaXt0r**
# To test
1. Chest in Diego's house is rotated right.
2. There are only 4 errors in log about "unknown Gothic version for world. Assuming G1" (Before that: ~140)
3. No log error about Culling

### CODE REVIEW

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs [hunk 1]
```diff
@@ -272,9 +278,12 @@ private static GameObject CreateFire(Fire vob, GameObject parent = null)
             if (vob.VobTree == "")
                 return go;
 
-            var vobTree = new ZenKit.World(GameData.Vfs, vob.VobTree);
+            if (!vobTreeCache.TryGetValue(vob.VobTree.ToLower(), out IWorld vobTree))
+            {
+                vobTree = new ZenKit.World(GameData.Vfs, vob.VobTree).Cache();
```
Comment (JucanAndreiDaniel) [inline Assets/GothicVR/Scripts/Creator/VobCreator.cs:283]:
```suggestion
                vobTree = new ZenKit.World(GameData.Vfs, vob.VobTree);
```

**Review by JucanAndreiDaniel — CHANGES_REQUESTED** [review]

**Comment by JaXt0r** [comment]
I'll also add, that we leverage latest ZenKit version which also solves the last error message.

File: Assets/GothicVR/Scripts/Creator/VobCreator.cs (hunk 1 above)
Comment (JaXt0r) [reply Assets/GothicVR/Scripts/Creator/VobCreator.cs:283]: We can keep this cache. At least the .zen itself isn't re-parsed every time.

**Comment by JaXt0r** [comment]
@JucanAndreiDaniel please test again. Unfortunately I couldn't see your fire again. But the errors are gone.
Please also check if the caching still makes sense. I think so, but 🤷

**Review by JucanAndreiDaniel — APPROVED** [review]
Make sure to restart unity as it might not let you see all the lights, i see no issue on my side

**Pull request merged**